CONTRIBUTING.md

```diff
@@ -1,51 +1,63 @@
- SHOULD NOT end with punctuation. Examples:
-  * Correct ✅: "unable to connect to server"
-  * Incorrect ❌: "Unable to connect to server"
-  * Incorrect ❌: "unable to connect to server."
 
-The following commands are available in the Makefile:
+### Makefile Targets
+
+You can use these commands in the Makefile:
 
 - `make build` compiles the code into executable binaries.
 - `make devtools` installs required development tools.
 - `make fmt` formats the code according to the Go standards.
-- `make check` runs various checks on the code, including formatting and linting.
-- `make test` performs all the tests including unit tests and system tests.
-- `make uint_test` performs only unit tests.
-- `make proto` generates [protobuf](https://protobuf.dev/) files, if you have made any changes to the proto buffer files.
+- `make check` runs checks on the code, including formatting and linting.
+- `make test` runs all the tests, including unit tests and system tests.
+- `make unit_test` runs only unit tests.
+- `make proto` generates [protobuf](https://protobuf.dev/) files.
+  Run this target if you have made any changes to the proto buffer files.
+
+### Error and Log Messages
+
+Error and log messages should not start with a capital letter (unless it's a proper noun or acronym) and
+should not end with punctuation.
 
-## CLI Guidelines
+#### Examples
 
-The help messages for CLI flags should follow this pattern:
+- Correct ✅: "unable to connect to server"
+- Incorrect ❌: "Unable to connect to server"
+- Incorrect ❌: "unable to connect to server."
 
-- Start all messages with a lowercase letter.
-- Avoid stating defaults in the help string, as Cobra automatically adds them.
-- Include a range for flags that accept a range of values.
+### Help Messages
+
+Follow these rules for help messages for CLI commands and flags:
+
+- Help string should not start with a capital letter.
+- Help string should not end with punctuation.
+- Don't include default value in the help string.
+- Include the acceptable range for the flags that accept a range of values.
 
 ## Commit Guidelines
 
-Please follow these guidelines when committing changes to Pactus blockchain:
+Please follow these rules when committing changes to the Pactus blockchain:
 
 - Each commit should represent a single, atomic change to the codebase.
   Avoid making multiple unrelated changes in a single commit.
-- Commit message should not be longer than 50 characters.
-- Commit message should start with lowercase and not ends with punctuation.
-- Write commit message in the imperative: "fix bug" and not "fixed bug" or "fixes bug".
-- Use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format for commit messages and Pull Request titles.
+- Use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format for commit messages and
+  Pull Request titles.
+
```

Comment:
Do we need to provide supported scopes here?

Reply:
These scopes may change in the future, so we provided a link to the linter (source of scopes).

Reply:
Got it. Thanks.